### DISCUSSION TITLE
Submit USPS images?

### DISCUSSION THREAD

**Discussion by sdholden28** [blakeblackshear/frigate#9909] [General]
Is there currently a way to submit USPS images to help with the model? I understand there's no object or label yet for USPS and it may be coming at some point. I just want to make sure I'm not missing an opportunity to contribute.

**Comment by NickM-27** [comment]
it is not possible yet

**Comment by TonySeeley** [comment]
Are there any plans to add this?  Seems like it would be pretty easy to add, just a new logo.

**Reply by NickM-27** [reply]
It's not really that easy, but an update with new labels is currently in the works. See https://www.reddit.com/r/frigate_nvr/comments/1dr535t/comment/lb28h9i/?utm_source=share&utm_medium=mweb3x&utm_name=mweb3xcss&utm_term=1&utm_content=share_button
